### ISSUE TITLE
Top 404s as of 2022/03/04

### ISSUE THREAD

**Issue by chalin** [kedacore/keda-docs#692]
Here are the top 404s:

> <img width="449" alt="image" src="https://user-images.githubusercontent.com/4140793/156805287-bada784e-bd05-44e8-891c-77340a9a35f0.png">

**Comment by chalin** [comment]
@tomkerkhove and @kedacore/keda-maintainers : please see my questions in the opening comment.

**Comment by chalin** [comment]
@tomkerkhove - another question: is @kedacore/keda-maintainers a private team? If so, could you make it public? Otherwise I can't refer to it. Thanks.

**Comment by tomkerkhove** [comment]
> * [ ]  `/v1alpha1` - @tomkerkhove

Ignore, that shouldn't be used

> * [ ]  `/docs/INSERT-CORRECT-VERSION/deploy/` - @tomkerkhove

Ignore, that shouldn't be used

> * [ ]  `/docs/2.6deploy` - [Fix deploy 404s #708](https://github.com/kedacore/keda-docs/pull/708)

Thanks

> * [ ]  `/scalers/mysql/`

Send to latest

> * [ ]  `/scalers/redis-lists/`

Send to latest

> * [ ]  `/%EF%BC%89%E3%80%82`

No idea what this is - I'd say ignore, that shouldn't be used

> * [ ]  `/scalers/apache-kafka/`

Send to latest

**Comment by chalin** [comment]
> > * [ ]   `/v1alpha1` - @tomkerkhove
> 
> Ignore, that shouldn't be used

Right, it shouldn't be used. But rather than a 404, it might be more helpful to someone visiting that link if it redirected to an explanation that `/v1alpha1` is just an API version ID -- possibly referencing https://kubernetes.io/docs/reference/using-api/#api-versioning? WDYT?

**Comment by tomkerkhove** [comment]
That's OK for me, but personally wouldn't spend much time on it given 404 is correct as well.

Otherwise we will have rules for all these scenarios
